src/routerpages/QuotePage.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+
+const QuotePage = () => {
+  const [quote, setQuote] = useState('');
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    const category = 'happiness';
+    const apiKey = '7zgpTKjAtUyiqNA2m0qzhA==35Yxa6v2sOTTNvVK';
```

Comment:
```suggestion
    const apiKey = import.meta.env.VITE_NINJAS_QUOTES_API_KEY;
```

Let's make sure we are not exposing any secret in the code, see https://vitejs.dev/guide/env-and-mode to learn more.

1. I would create a `.env` file added to `.gitignore` locally
2. I would create this same env variable on netlify env variables settings to be used by the deplyed app